frameworks/JavaScript/express/benchmark_config.json

```diff
@@ -60,6 +60,66 @@
       "display_name": "express",
       "notes": "",
       "versus": "node"
+    },
+    "iojs": {
+      "setup_file": "setup_iojs",
+      "json_url": "/json",
+      "plaintext_url": "/plaintext",
+      "port": 8080,
+      "approach": "Realistic",
+      "classification": "Micro",
+      "database": "None",
+      "framework": "express",
+      "language": "JavaScript",
+      "orm": "Full",
+      "platform": "iojs",
+      "webserver": "None",
+      "os": "Linux",
+      "database_os": "Linux",
+      "display_name": "express",
+      "notes": "",
+      "versus": "node"
+    },
+    "mongodb-iojs": {
+      "setup_file": "setup_iojs",
+      "db_url": "/mongoose",
+      "query_url": "/mongoose?queries=",
+      "update_url": "/mongoose-update?queries=",
+      "port": 8080,
+      "approach": "Realistic",
+      "classification": "Micro",
+      "database": "MongoDB",
+      "framework": "express",
+      "language": "JavaScript",
+      "orm": "Full",
+      "platform": "iojs",
+      "webserver": "None",
+      "os": "Linux",
+      "database_os": "Linux",
+      "display_name": "express",
+      "notes": "",
+      "versus": "node"
+    },
+    "mysql-iojs": {
+      "setup_file": "setup_iojs",
+      "db_url": "/mysql-orm",
+      "query_url": "/mysql-orm?queries=",
+      "fortune_url": "/fortune",
+      "update_url": "/mysql-orm-update?queries=",
+      "port": 8080,
+      "approach": "Realistic",
+      "classification": "Micro",
+      "database": "MySQL",
+      "framework": "express",
+      "language": "JavaScript",
+      "orm": "Full",
+      "platform": "iojs",
+      "webserver": "None",
+      "os": "Linux",
+      "database_os": "Linux",
+      "display_name": "express",
```

Comment:
No, we render the platform on the results website already - let's leave this as "express".